postgres/src/schema.rs

```diff
@@ -96,6 +133,20 @@ impl TableSchema {
     pub fn has_primary_keys(&self) -> bool {
         self.column_schemas.iter().any(|cs| cs.primary)
     }
+
+    /// Compares two [`TableSchema`] instances, excluding the [`ColumnSchema`]'s `nullable` field.
+    ///
+    /// Return `true` if all fields except `nullable` are equal, `false` otherwise.
+    pub fn partial_eq(&self, other: &TableSchema) -> bool {
```

Comment:
It would be nice if we could find a solution for this and actually either remove `nullable` or figure out how to get it from the replication protocol.